public/v4/apps/homepage.yml

```diff
@@ -0,0 +1,49 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: ghcr.io/gethomepage/homepage:$$cap_homepage_version
+        environment: null
+        PUID: $$cap_puid
+        PGID: $$cap_guid
+        volumes:
+            - $$cap_config_path:/app/config
+            - /var/run/docker.sock:/var/run/docker.sock:ro
+        restart: always
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_puid
+          label: User ID
+          defaultValue: '1000'
+          description:
+              User ID that the process uses, run (id $user) in your instance to
+              see the id
+          validRegex: /.{1,}/
+        - id: $$cap_guid
+          label: Group ID
+          defaultValue: '1000'
+          description:
+              Group ID that the process uses, run (id $user) in your instance to
+              see the id
+          validRegex: /.{1,}/
+        - id: $$cap_homepage_version
+          label: Homepage Version
+          defaultValue: v0.7.4
+          description: Check out their GitHub packages page for the valid tags
+              https://github.com/gethomepage/homepage/releases
+          validRegex: /^([^\\s^\\/])+$/
+        - id: $$cap_config_path
+          label: Configuration Path
+          defaultValue: /path/to/config
```

Comment:
Why don't use a volume name. It's more common and you don't need to make sure that the path exists

Reply:
done! still works. `https://test2.jaw.dev/`